### PR TITLE
Create Configuration User Stories as Gherkin Specs

### PR DESCRIPTION
**Pull request by nicolepeverley**
This PR creates feature files for configuration user stories for syncing contacts with SaaSquatch and participants with HubSpot.

https://trello.com/c/bA4ITRGM/58-cucumber-specs-for-configuration

### CODE REVIEW

File: features/SaaSquatch-to-Hubspot/Sync Participants with HubSpot.feature [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        Then the contact is deleted in HubSpot
+
+    Scenario: Do not delete contact in HubSpot when the participant is deleted in SaaSquatch
+        Given a participant is deleted in SaaSquatch
+        When I have toggled the delete contact option off
+        Then the contact is not deleted in HubSpot
+
+    Scenario: Import existing referral links to contact in HubSpot from participant in SaaSquatch
+        Given a contact exists in HubSpot and the particpant exists in SaaSquatch
+        When I have toggled the import existing referral links option on and selected what referral programs to sync
```
Comment (nicolepeverley) [inline features/SaaSquatch-to-Hubspot/Sync Participants with HubSpot.feature]: @EvanDungate do we know if we can check what referral programs a participant belongs to in SaaSquatch easily from the API?

**Review by ChristopherRDunn — APPROVED** [review]
Looks great

**Review by lucianojd — APPROVED** [review]
Looks good. For testing I think we should consider moving this file into `/frontend/features/`.

**Review by MCunningham211 — APPROVED** [review]
This looks good, We will have to find out from backend whether deletion is possible in both directions.

**Pull request merged**